recipes/setup.rb

```diff
@@ -197,3 +197,9 @@
 
   fire_hook(:setup, items: databases + [source, framework, appserver, worker, webserver])
 end
+
+# setup hooks for appservers and workers may need to reload monit configs
+execute 'monit reload' do
+  action :nothing
+  only_if 'which monit'
```

Comment:
I think the opsworks agent may install monit for its own purposes, so maybe we can drop the conditional check for monit and maybe just make it a full blown dependency in the metadata file.